src/locales/ar/texts.json

```diff
@@ -0,0 +1,376 @@
+    "loadApiItemsFailed": "تعذر تحميل عناصر API.",
+    "loadFilteredItems": "عذرًا، فشل تحميل قائمة العناصر المصفاة.",
+    "loadItems": "عذرًا، فشل تحميل قائمة عناصر STAC.",
+    "networkError": "قد تحدث هذه المشكلة عندما لا تسمح الخوادم بالوصول الخارجي عبر متصفحات الويب (مثل عدم وجود رؤوس CORS). @:errors.contactProvider",
+    "noExternalAccess": "الوصول إلى الكتالوجات الخارجية غير مسموح!",
+    "notFound": "المورد المطلوب غير موجود. @:errors.contactProvider",
+    "serverError": "واجه الخادم مشكلة. @:errors.contactProvider",
+    "unauthorized": "يفتقر الطلب إلى بيانات الاعتماد، مثل رمز API. يرجى تقديم بيانات الاعتماد والمحاولة مرة أخرى."
+  },
+  "featureExperimental": "هذه الميزة لا تزال تجريبية وقد تعطي نتائج غير متوقعة!",
+  "fullscreen": {
+    "exit": "الخروج من وضع ملء الشاشة",
+    "show": "عرض بملء الشاشة"
+  },
+  "goBack": {
+    "description": "العودة إلى {type}",
+    "label": "رجوع"
+  },
+  "goToCollection": {
+    "description": "الانتقال إلى المجموعة المرتبطة",
+    "descriptionWithTitle": "@:goToCollection.description: {title}",
+    "label": "مجموعة"
+  },
+  "goToParent": {
+    "description": "الانتقال إلى الأصل",
+    "descriptionWithTitle": "@:goToParent.description: {title}",
+    "label": "أعلى"
+  },
+  "in": "في {catalog}",
+  "index": {
+    "api": "واجهة برمجة التطبيقات",
+    "catalog": "الكتالوج",
+    "load": "تحميل",
+    "selectStacIndex": "... أو اختر واحدًا من {stacIndex}:",
+    "specifyCatalog": "يرجى تحديد كتالوج STAC أو API...",
+    "urlInvalid": "عنوان URL غير صالح.",
+    "urlMissingHost": "عنوان URL يفتقد إلى المضيف.",
+    "urlMissingProtocol": "عنوان URL يفتقد إلى البروتوكول."
+  },
+  "items": {
+    "filter": "عوامل التصفية",
+    "hideFilter": "إخفاء عوامل التصفية",
+    "noTime": "لم يتم تحديد وقت",
+    "noneAvailableForCollection": "لا توجد عناصر متاحة لهذه المجموعة.",
+    "showFilter": "إظهار عوامل التصفية"
+  },
+  "leaflet": {
+    "close": "إغلاق",
+    "noFeatureProperties": "لا توجد بيانات إضافية متاحة لهذه الميزة.",
+    "stayLayer": {
+      "error": "عذرًا، فشل إضافة البيانات إلى الخريطة."
+    },
+    "zoom": {
+      "in": {
+        "description": "تكبير",
+        "label": "+"
+      },
+      "out": {
+        "description": "تصغير",
+        "label": "-"
+      }
+    }
+  },
+  "loading": "جارٍ التحميل...",
+  "map": "الخريطة",
+  "messageForSchemaError": "{message}، للمخطط {schemaPath}",
+  "metadata": {
+    "general": "عام",
+    "title": "البيانات الوصفية"
+  },
+  "multiselect": {
+    "andMore": "و {count} المزيد",
+    "deselectLabel": "اضغط على Enter للإزالة",
+    "keywordsPlaceholder": "اختر الكلمات المفتاحية",
+    "placeholder": "اختر خيارًا",
+    "selectLabel": "اضغط على Enter للاختيار",
+    "selectedLabel": "تم الاختيار"
+  },
+  "open": "Open",
+  "pagination": {
+    "first": "« First",
+    "last": "Last »",
+    "next": "Next ›",
+    "previous": "‹ Previous"
+  },
+  "poweredBy": "Powered by {link}",
```

Comment:
Translations missing. Untranslated phrases should be removed from the file until they are translated.